assets/src/edit-story/components/checklist/index.js

```diff
@@ -77,21 +93,32 @@ export function Checklist() {
   return (
     <DirectionAware>
       <ChecklistCountProvider>
-        <Wrapper ref={wrapperRef}>
-          <Popup
-            popupId={POPUP_ID}
-            isOpen={isOpen}
-            ariaLabel={__('Checklist', 'web-stories')}
-          >
+        <Wrapper role="region" aria-label={CHECKLIST_TITLE} ref={wrapperRef}>
+          <Popup popupId={POPUP_ID} isOpen={isOpen} ariaLabel={CHECKLIST_TITLE}>
             <NavigationWrapper ref={navRef}>
               <TopNavigation
                 onClose={() => setIsOpen(false)}
-                label={__('Checklist', 'web-stories')}
+                label={CHECKLIST_TITLE}
                 popupId={POPUP_ID}
               />
-              <PriorityChecks />
-              <DesignChecks />
-              <AccessibilityChecks />
+              <Tablist aria-label={__('Tab list of issues', 'web-stories')}>
```

Comment:
Don't love this aria label. Any better ideas? This is the tablist that contains all 3 sections in our checklist.

Reply:
Maybe `Potential Story issues by category`? Then the tabs themselves have their own title which gives context but doesn't repeat info?